### PR TITLE
Release `0.7.1`

### PR DESCRIPTION
**Pull request by gareth-j**
* **Summary of changes** (Bug fix, feature, docs update, ...)

This is a patch release before we do the bigger `0.8.0` release.

### Fixed

- Missing requirement for filelock package added to conda environment file [PR #857](https://github.com/openghg/openghg/pull/857)
- Missing store argument adding to search function allow searching within specific object stores [PR #859](https://github.com/openghg/openghg/pull/859)
- Bug fix for conversion of species parameter with its synonym value inside get_obs_surface_local. [PR #871](https://github.com/openghg/openghg/pull/871)
- Bug fix for allowing a period to be specified when this cannot be inferred from the input data [PR #899](https://github.com/openghg/openghg/pull/899)

### CODE REVIEW

**Comment by rt17603** [comment]
Do we want to get the fix for #937 into `devel` before we do this patch release?

**Comment by gareth-j** [comment]
> Do we want to get the fix for #937 into `devel` before we do this patch release?

Yeah, we'll merge that before this goes in

**Comment by rt17603** [comment]
Sorry I meant #935 - the `test_standardise_obs_openghg` test

File: openghg/dataobjects/_datamanager.py [hunk 1]
```diff
@@ -273,6 +273,6 @@ def data_manager(data_type: str, store: str, **kwargs: Dict) -> DataManager:
     if store not in writable_stores:
         raise ObjectStoreError(f"You do not have permission to write to the {store} store.")
 
-    res = search(data_type=data_type, **kwargs)
+    res = search(data_type=data_type, store=store, **kwargs)
```
Comment (rt17603) [inline openghg/dataobjects/_datamanager.py:276]: This seems to have been an additional bug fix: #859 which should be added to the CHANGELOG.md

File: environment.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ dependencies:
   - addict
   - dask
   - h5netcdf
+  - filelock
```
Comment (rt17603) [inline environment.yaml:12]: Another small update to add to the CHANGELOG.md: #857

**Review by rt17603 — CHANGES_REQUESTED** [review]
I believe there are 2 additional PR fixes to add to the CHANGELOG.md details but other than that this looks good to go. Tests pass for me locally.

**Review by rt17603 — APPROVED** [review]

**Review by milliams — APPROVED** [review]

**Pull request merged**